### PR TITLE
fix: Garbled filename when unzipped

### LINKED ISSUE #680: Garbled filename when unzipped
## Describe the bug
When I download a script, I will get garbled filename/folder in my script directory.

## To Reproduce

1. Go to `プラグイン&スクリプト` tab
2. Click on `AviUtlスクリプト一式`
3. Press `Download` button in BowlRoll's page
4. The message shows `インストール完了`

## Expected behavior
For example, in `./script/ANM_ssd` folder, the correct filename is `2値化ノイズ.anm`, but I got `2.anm` instead.

## Screenshots
In `./script/ANM_ssd`:
圖片
The correct filename is:
圖片

## Desktop:

- OS: Windows 10
- OS Code Page: 950(chinese)
- Version: 3.2.0

## Additional context
The problem might caused by **Wrong Encoding Code Page**. When I use 7zip to unzip script.zip, I also got garbled filename/folder. But I can get correct result with code page 932(Shift JIS).

I tried to fix the problem in `src/lib/unzip.js` line 38:
```js
  const zipStream = extractFull(zipPath, targetPath, {
    $bin: process.platform === 'win32' ? pathTo7zipWin : pathTo7zipCross,
    overwrite: 'a',
    method: ['cp=932'], // https://sevenzip.osdn.jp/chm/cmdline/switches/method.htm
  });
```
But the result is not good. Both garbled file and correct file are in the folder:
圖片

AviUtl is a very good software, but installing and setting is quiet annoying to me. APM solve the problem, so I really love the this project. 
P.S. Sorry for the bad grammar. I am not good at English and Japanese.

### PR DESCRIPTION
**Pull request by FOBshippingpoint**
## Description
- Change code page to 932(Shift_JIS).
- Refactor: simplify pathTo7zip settings.

## Related Issue
- fix #680 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/lib/unzip.js [hunk 1]
```diff
@@ -4,13 +4,12 @@ import path from 'path';
...
+const pathTo7zip = process.platform === 'win32' ? win7zip['7z'] : path7za;
+
 // https://github.com/puppeteer/puppeteer/issues/2134#issuecomment-408221446
-const pathTo7zipCross = isDevEnv
-  ? path7za
-  : path7za.replace('app.asar', 'app.asar.unpacked');
-const pathTo7zipWin = isDevEnv
-  ? win7zip['7z']
-  : win7zip['7z'].replace('app.asar', 'app.asar.unpacked');
+if (isDevEnv) {
```
Comment (mitosagi) [inline src/lib/unzip.js:10]: Probably inverted.

**Review by mitosagi — CHANGES_REQUESTED** [review]
I just want to confirm one thing.

File: src/lib/unzip.js (hunk 1 above)
Comment (FOBshippingpoint) [reply src/lib/unzip.js:10]: Yes, I will fix this.

**Review by mitosagi — APPROVED** [review]
Looks good to me.

File: src/lib/unzip.js [hunk 2]
```diff
@@ -4,13 +4,12 @@ import path from 'path';
...
+
 // https://github.com/puppeteer/puppeteer/issues/2134#issuecomment-408221446
-const pathTo7zipCross = isDevEnv
-  ? path7za
-  : path7za.replace('app.asar', 'app.asar.unpacked');
-const pathTo7zipWin = isDevEnv
-  ? win7zip['7z']
-  : win7zip['7z'].replace('app.asar', 'app.asar.unpacked');
+if (!isDevEnv) {
+  pathTo7zip.replace('app.asar', 'app.asar.unpacked');
```
Comment (hal-shu-sato) [inline src/lib/unzip.js:11]: `String.replace()` does not change the object.
It just returns a new string.
```suggestion
  pathTo7zip = pathTo7zip.replace('app.asar', 'app.asar.unpacked');
```

File: src/lib/unzip.js [hunk 3]
```diff
@@ -4,13 +4,12 @@ import path from 'path';
 import win7zip from 'win-7zip';
 const isDevEnv = process.env.NODE_ENV === 'development';
 
+const pathTo7zip = process.platform === 'win32' ? win7zip['7z'] : path7za;
```
Comment (hal-shu-sato) [inline src/lib/unzip.js:7]:
```suggestion
let pathTo7zip = process.platform === 'win32' ? win7zip['7z'] : path7za;
```

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
I have one point to review.

File: src/lib/unzip.js (hunk 2 above)
Comment (FOBshippingpoint) [reply src/lib/unzip.js:11]: Sorry for this stupid mistake. I fixed the problem. I will pay more attention next time.

Comment (hal-shu-sato) [reply src/lib/unzip.js]: Please do not say such a thing.
We, everyone, could make a mistake.
It is OK.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for fixing it and contributing to apm!

**Pull request merged**